src/components/settings/SettingsLMSTab/LMSConfigs/CornerstoneConfig.jsx

```diff
@@ -104,5 +123,10 @@ const CornerstoneConfig = ({ id, onClick }) => {
 CornerstoneConfig.propTypes = {
   id: PropTypes.string.isRequired,
   onClick: PropTypes.func.isRequired,
+  existingData: PropTypes.shape({
+    id: PropTypes.number,
+    cornerstone_base_url: PropTypes.string,
```

Comment:
hmm are these fields name not being camelCased before processing by these UI components? it may be a good thing to do that to keep with JS style? Alternatively you can leave it like this here, but map them to camelCased values in components code ... but ideally we camel case before we even pass to these components (just a thought we have done this in other places so ... )

Reply:
the reason it's snake cased is because existingData comes directly from the API response. ESLint seems to be fine with how it is, but I can look into a util